static/templates/settings/dropdown_list_widget.hbs

```diff
@@ -6,8 +6,23 @@
         </label>
         {{/if}}
         <button class="button rounded dropdown-toggle" type="button" data-toggle="dropdown">
+            {{!-- TODO(compose-dropdown)
+            * make this appear only for the two relevant dropdowns
+            * add to settings.js to toggle icons in settings menu (enable it there in a separate commit)
+            * i probably want to put this outside of this file since it's pretty use-specific
+            --}}
+            <span id="compose-lock-icon">
+                <i class="fa fa-lock" title="{{t 'This is a private stream' }}" aria-hidden="true"></i>
+            </span>
+            <span id="compose-globe-icon">
+                <i class="zulip-icon zulip-icon-globe" title="{{t 'This is a web-public stream' }}" aria-hidden="true"></i>
+            </span>
             <span id="{{widget_name}}_name"></span>
-            <i class="fa fa-chevron-down"></i>
+            {{#if dropup }}
+                <i class="fa fa-chevron-up"></i>
+            {{else}}
+                <i class="fa fa-chevron-down"></i>
+            {{/if}}
```

Comment:
I think this probably live in a separate template; from an abstractions standpoint, `compose` shouldn't appear in the `dropdown_list_widget` codebase at all.

Reply:
I totally agree! Just gotta wrap my head around how to set that up. I remember we were talking about it on Zulip a while ago.